jobs/harbor/spec

```diff
@@ -216,3 +217,6 @@ properties:
   registry_storage_provider.nfs.mount_point:
     description: "The local mount point for remote NFS Server."
     default: "/harbor-registry"
+  enable_externalssl:
```

Comment:
Since it's a configuration setting for the docker registry server, we can rename it as `registry.enable_external_ssl`.